android/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 31
+    compileSdk 34
```

Comment:
I'm wondering if this was needed or if it was only on the `purchases_ui_flutter` library... However, I think it shouldn't hurt.

Reply:
I reverted this in my followup commits